dandi/dandiapi.py

```diff
@@ -937,6 +937,29 @@ def from_data(cls, client: DandiAPIClient, data: dict[str, Any]) -> RemoteDandis
             client=client, identifier=data["identifier"], version=version, data=data
         )
 
+    @staticmethod
+    def _normalize_path(path: str) -> str:
+        """
+        Helper to normalize path before passing it to the server.
+
+        We and API call it "path" but it is really a "prefix" with inherent
+        semantics of containing directory divider '/' and emphasizing with
```

Comment:
I don't think "emphasizing" is the word you want here.  Maybe "and referring to a directory when terminated with '/'."